code/game/objects/items/weapons/grenades/frag.dm

```diff
@@ -71,3 +71,32 @@
 	blast_radius = 0
 	damage_step = 8
 	spread_range = 7
+
+/obj/item/grenade/frag/white_phosphorous
+	name = "SA WPG \"Geneva\""
```

Comment:
```suggestion
	name = "SA WPG \"Basel\""
```
Named after a different Switzerland city

Reply:
Unbaseled and soulless.

Reply:
Only oldNT ones should be called by Swiss names, I'd suggest using a serbian name for this one